dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -197,44 +187,38 @@ def prepare_airflow_packages(
     name="prepare-provider-documentation",
     help="Prepare CHANGELOG, README and COMMITS information for providers.",
 )
-@option_verbose
-@option_dry_run
+@option_debug_release_management
+@argument_packages
 @click.option(
     "--base-branch",
     type=str,
     default="main",
 )
 @option_github_repository
+@option_verbose
+@option_dry_run
 @option_answer
-@option_debug_release_management
-@argument_packages
 def prepare_provider_documentation(
-    verbose: bool,
-    dry_run: bool,
-    base_branch: str,
     github_repository: str,
-    answer: str | None,
+    base_branch: str,
     debug: bool,
     packages: list[str],
 ):
-    perform_environment_checks(verbose=verbose)
+    perform_environment_checks()
     shell_params = ShellParams(
-        verbose=verbose,
         mount_sources=MOUNT_ALL,
         github_repository=github_repository,
         python=DEFAULT_PYTHON_MAJOR_MINOR_VERSION,
-        answer=answer,
         base_branch=base_branch,
         skip_environment_initialization=True,
     )
-    rebuild_or_pull_ci_image_if_needed(command_params=shell_params, dry_run=dry_run, verbose=verbose)
+    rebuild_or_pull_ci_image_if_needed(command_params=shell_params)
     cmd_to_run = ["/opt/airflow/scripts/in_container/run_prepare_provider_documentation.sh", *packages]
+    answer = get_forced_answer()
     result_command = run_with_debug(
         params=shell_params,
         command=cmd_to_run,
-        enable_input=not answer or answer.lower() not in ["y", "yes"],
-        verbose=verbose,
-        dry_run=dry_run,
+        enable_input=answer is None or answer.lower() not in ["y", "yes"],
```

Comment:
Consider the same formula you sued elsewhere?

```suggestion
        enable_input=answer is None or answer[0].lower() not in "y",
```